### PR TITLE
Add peer_storage_path to autopeering config

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Review by grtlr — APPROVED** [review]

File: bee-network/bee-autopeering/src/config.rs [hunk 1]
```diff
@@ -96,6 +99,11 @@ impl AutopeeringConfig {
         self.entry_nodes
     }
 
+    /// The peer storage path.
+    pub fn peer_storage_path(&self) -> &Path {
```
Comment (Alex6323) [inline bee-network/bee-autopeering/src/config.rs:103]: why `peer_storage_path` and not `peer_store_path`?

**Review by Adam-Gleave — APPROVED** [review]

File: bee-network/bee-autopeering/src/config.rs (hunk 1 above)
Comment (Adam-Gleave) [reply bee-network/bee-autopeering/src/config.rs:103]: `peer_store_path` sounds odd to me -- that could be the path that peers use to store something.

Comment (thibault-martinez) [reply bee-network/bee-autopeering/src/config.rs:103]: To be consistent with the other storage that shares the same root. And store is weird, it really is a storage.

Comment (Alex6323) [reply bee-network/bee-autopeering/src/config.rs:103]: @Adam-Gleave well, it used to be `peerstore` but someone (:wink: ) wanted the underscore. Would `peerstore_path` be okay?

Comment (thibault-martinez) [reply bee-network/bee-autopeering/src/config.rs:103]: That wouldn't be rust nor TOML compliant

**Pull request merged**